src/metrics/index.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const HttpClient = require("@actions/http-client").HttpClient
+const AmbassadorClient = require('../ambassador/api-client');
+const core = require('@actions/core');
+const configure = require('../configure');
+
+const httpClient = new HttpClient();
+const ambassadorClient = new AmbassadorClient(process.env.TELEPRESENCE_API_KEY)
+
+class MetritonClient {
+    static async sendMetricsReport(action){
+        try {
+            const reportDestination = 'https://metriton.datawire.io/beta/scout'
+            const applicationName = 'telepresence-github-action-integration'
+            const installId = fs.readFileSync(`${configure.getTelepresenceConfigPath()}/id`).toString()
+            core.info('install id: ' + installId)
```

Comment:
This messages go to the summary section of the GH actions job execution. If we want to keep it we should be explicit about what's this id.

```suggestion
            core.info('Telepresence install id: ' + installId)
```

Reply:
This is just a log not being removed